twoliter/src/variant.rs

```diff
@@ -0,0 +1,46 @@
+use crate::common::exec;
+use anyhow::{Context, Result};
+use serde_json::Value;
+use std::path::Path;
+use tokio::process::Command;
+
+pub(crate) async fn variant_version<S>(variants_dir: &Path, variant: S) -> Result<String>
```

Comment:
I think we should execute this cargo command inside the SDK container because eventually we want #114. This command is an easy one to run inside docker because it doesn't cause `build.rs` to be executed (which would create another container).

Ultimately this doesn't matter so much right now, but it would be one less thing to change later.

Reply:
We will work on this later.